### PR TITLE
New Case Study: Deploying Flynn Cluster on Azure

### PR DESCRIPTION
**Pull request by felixrieseberg**
Added a new DevOps case study describing how to deploy a Flynn cluster for 12-factor-apps on Azure. I'd appreciate a quick review from the usual devops heros, maybe @timfpark or @ritazh.

### CODE REVIEW

File: _posts/2015-08-25-flynn-azure-cluster.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+![Flynn Installing a Cluster]({{site.baseurl}}/images/2015-08-25-flynn-azure-cluster/flynn2.png)
+
+Depending on how big your cluster is, installation will take a while. You can either wait for it to complete or read on to learn about how apps are being deployed.
+
+###### Issues
+Flynn is still in active development, meaning that things can still very much break. In my case, I had to attempt installation a total of three times before my cluster spun up.
+
+### Congrats, you're running Flynn!
+Flynn can run virtually anything, but it's especially good at running Go, Java, Node.js, PHP, Python, and Ruby. If you're already familiar with the incredible convenience of Heroku buildpacks, you'll be happy to hear that Flynn just uses those. If you're not, think of buildpacks like packages that contain your application, describing the environment and dependencies for your app. They are automatically created as soon as you push a repository/app to Flynn.
```
Comment (timfpark) [inline _posts/2015-08-25-flynn-azure-cluster.md]: Consider moving up the https://flynn.io/docs link here as a jump off point for the next steps on getting started deploying an app.

**Comment by timfpark** [comment]
I ran through the instructions and created a Flynn cluster - looks great to me - love the walkthrough - I would have for sure stubbed my toe on the complex AAD setup without that.

One question:  Why are only D series VMs an option and not also A series?

Also, my cluster create failed with: 

Running `sudo flynn-host collect-debug-info` on 40.76.91.37
t=2015-08-19T19:56:46+0000 lvl=info msg="uploading logs and debug information to a private, anonymous gist"
t=2015-08-19T19:56:46+0000 lvl=info msg="this may take a while depending on the size of your logs"
t=2015-08-19T19:56:46+0000 lvl=info msg="getting flynn-host logs"
t=2015-08-19T19:56:46+0000 lvl=info msg="getting job logs"
t=2015-08-19T19:56:46+0000 lvl=info msg="getting system information"
t=2015-08-19T19:56:46+0000 lvl=info msg="creating anonymous gist"
t=2015-08-19T19:56:46+0000 lvl=info msg="debug information uploaded to: https://gist.github.com/3d2f7bf901aeb471ad34"
Error: bootstrap: controller run-app error: resource: unexpected status code 500

I will try again per your notes that it is still pretty fragile - but the Azure integration seemed to all work great.

LGTM

File: _posts/2015-08-25-flynn-azure-cluster.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+author-link: "http://www.felixrieseberg.com"
+#author-image: "{{ site.baseurl }}/images/FelixRieseberg/photo.jpg" //should be square dimensions
+date:   2015-08-30 10:00:00
+categories: Azure DevOps Flynn
+color: "blue"
+#image: "{{ site.baseurl }}/images/imagename.png" #should be ~350px tall
+excerpt: "Deploying Flynn Clusters on Azure"
+---
+
+Flynn is a next-generation PaaS (Platform as a Service) solution that can deploy, run, and scale both twelve-factor apps as well as stateful services. The company behind Flynn (which is also named Flynn) recently joined Y Combinator and is moving fast – and has bold goals: “Flynn deploys apps, scales apps, and manages databases with best practices right out of the box. Automatically doing all the things that were too complicated, expensive, or time consuming to do manually. And because it's open source, modular, and built on a set of core APIs, it's easy to extend or customize to any environment or need. Flynn is simply the easiest, most powerful way to run and scale your applications”.
```
Comment (stefangordon) [inline _posts/2015-08-25-flynn-azure-cluster.md]: "and is moving fast - and has bold goals" is the only text in the case study I stumbled on.  I'd probably just change it to "and is moving fast with bold goals" for readability (I acknowledge it isn't as impactful).

**Comment by stefangordon** [comment]
Well written, great work.  You properly hook the right range of users at the top, and provide all the technical detail below.  I'm testing the process right now, but the case study LGTM.

**Comment by ritazh** [comment]
This is very nice Felix. Great walk through. One thing that might be helpful is to add some errors and troubleshooting steps since you mentioned that you tried three times to get this to work. I think it would be helpful if others run into similar issues.
LGTM

File: _posts/2015-08-25-flynn-azure-cluster.md (hunk 2 above)
Comment (felixrieseberg) [inline _posts/2015-08-25-flynn-azure-cluster.md]: Thanks!

File: _posts/2015-08-25-flynn-azure-cluster.md (hunk 1 above)
Comment (felixrieseberg) [inline _posts/2015-08-25-flynn-azure-cluster.md]: Thanks!

**Comment by felixrieseberg** [comment]
Thanks for the review, @timfpark, @stefangordon, @ritazh! I incorporated all your proposed changes.

**Pull request merged**